diary-alexander-jaffray/modernized/38-notes-memoirs.adoc

```diff
@@ -795,10 +834,17 @@ And do not think the time long, for all time is in the Father`'s hand, his power
 and therefore keep the word of patience, and exercise that gift,
 and the Lord strengthen you in your sufferings, in his holy spirit of faith, amen!
 
+[.signed-section-signature]
 G+++.+++ F.
 
+--
+
+[.centered]
 === Note P.
 
+[.embedded-content-document.address]
+--
+
 A Seasonable Warning and serious exhortation to,
```

Comment:
This little chunk, starting with "A Seasonable Warning..." etc I think would look better as a blurb heading, or something like that. Both here and in the original. 👍🏻